docs/modules/reference/pages/configuration/filters/filters.adoc

```diff
@@ -0,0 +1,68 @@
+
+[[filters]]
+= Filters
+
+A filter (also known as a rule) is a shorthand expression that allows you to select IP addresses from the ipinterface table.
+The rule builds the WHERE clause of a SQL query. 
+The https://github.com/OpenNMS/opennms/blob/develop/opennms-config/src/main/resources/database-schema.xml[database-schema.xml] file in the `/opt/opennms/etc` directory informs the filter parser which tables can be used in an expression.
+
+== Explanation of database-schema.xml file
+
+Each table the filter code uses appears in a `<table>` tag. 
+If a table has an attribute of "visible=false", then none of the columns in that table can be used in the WHERE clause and thus cannot appear in the rule. 
```

Comment:
Quotes here look wrong to me, but I don't know what actually conforms to style.
```suggestion
If a table has an attribute of `visible=false`, then none of the columns in that table can be used in the WHERE clause and thus cannot appear in the rule. 
```

Reply:
In this case you are correct. I didn't consider this as a literal attribute, which is why I did the double-quotes. Since it is we want it to appear the way you've corrected it.